make/__init__.py

```
@@ -0,0 +1,422 @@
+            os.unlink(DIGEST_CACHE_PATH)
+
+
+def _remote_image_digest(image: str, skip_manifest: bool = False) -> str:
+    e: Exception | None = None
+    for attempt in range(10):
+        assert image_exists(image, True, skip_manifest), (
+            f"{image} does not exist on remote server"
+        )
+        try:
+            digest = image_digest(image, True)
+            return digest
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)